File: 1180.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
	int n,position=0,menor;
	int *vet;

	cin >> n;

	vet = (int*)malloc(sizeof(int)*n);
	for(int i=0;i<n;i++) cin >> vet[i];
	menor = vet[0];
	for(int i=0;i<n;i++){
		if(vet[i]<menor){
			menor = vet[i];
			position = i;
		}
	}

	cout << "Menor valor: " << menor << endl;
	cout << "Posicao: " << position << endl;

	return 0;
}

File: 1180.py
n = int(input())
entrada = [int(i) for i in input().split()]
menor = entrada[0]
position,cont = 0,0
for j in range (n):
	if entrada[j] < menor:
		menor = entrada[j]
		position = cont
	cont += 1
print("Menor valor: %d" % menor)
print("Posicao: %d" % position)
